lib/index.js

```diff
@@ -12,17 +12,17 @@ module.exports.getInfo = function(source, callback) {
       info[key] = _info[key];
     });
 
-    if (info.vector_layers) {
-      info.format = "pbf";
-    }
-
     info.name = info.name || "Untitled";
     info.center = info.center || [-122.4440, 37.7908, 12];
     info.bounds = info.bounds || [-180, -85.0511, 180, 85.0511];
     info.format = info.format || "png";
     info.minzoom = Math.max(0, info.minzoom | 0);
     info.maxzoom = info.maxzoom || Infinity;
 
+    if (info.vector_layers) {
+      info.format = "pbf";
```

Comment:
Nice. I think this fixes a problem I was running into in the past where `info.format` was `undefined` for some MBTiles containing PBFs.